tensorboard/plugins/base_plugin.py

```diff
@@ -196,3 +196,10 @@ def __init__(self, plugin_class):
 
   def load(self, context):
     return self._plugin_class(context)
+
+
+class FlagsException(ValueError):
```

Comment:
User-defined errors in Python have names ending in `Error`, not
`Exception` (just like the built-in exceptions). Rename to `FlagsError`?

Reply:
Done